src/server/routes/legacyRoutes.ts

```diff
@@ -92,23 +102,12 @@ export const fetchStateV1Route: Serverlet<ApiRequest> = async request => {
  * Response body: unused
  */
 export const attachUserV1Route: Serverlet<ApiRequest> = async request => {
-  const { connection, date, log, query } = request
+  const { connection, date, query } = request
 
   const checkedQuery = checkPayload(asAttachUserQuery, query)
   if (checkedQuery.error != null) return checkedQuery.error
   const { deviceId, userId } = checkedQuery.clean
 
-  const device = await Device.fetch(deviceId)
-  if (device == null) {
-    return errorResponse(`Cannot find device ${deviceId}`, { status: 404 })
```

Comment:
will the v2 update throw if deviceId isn't found in the new models?

Reply:
The v2 `getDeviceById` will return an empty fallback device if the lookup fails, so no, it won't cause a problem.